docker/mockoon/data.ts

```
@@ -0,0 +1,52 @@
+    if (!fs.existsSync(OUTPUT_DIR)) {
+      fs.mkdirSync(OUTPUT_DIR, { recursive: true });
+    }
+
+    // Save response as JSON
+    const data: string = await response.text();
+    fs.writeFileSync(OUTPUT_FILE, data, 'utf-8');
+
+    console.log(`Schema successfully saved to: ${OUTPUT_FILE}`);
+  } catch (error) {
+    if ((error as Error).name === 'AbortError') {
+      console.error('Schema fetch timeout after 5 seconds');
```

ESLint

[error] Unexpected console statement.

(no-console)